test/gateways/authorize_net_test.exs

```diff
@@ -136,24 +166,24 @@ defmodule Gringotts.Gateways.AuthorizeNetTest do
     test "successful response with right params" do
       with_mock HTTPoison,
       [request: fn(_method, _url, _body, _headers) -> MockResponse.successful_refund_response end] do
-        {:ok, response} = ANet.refund(@amount, @refund_id, @opts_refund)
-        assert response.raw["createTransactionResponse"]["messages"]["resultCode"] == "Ok"
+        assert {:ok, response} = ANet.refund(@amount, @refund_id, @opts_refund)
+        assert response.params["createTransactionResponse"]["messages"]["resultCode"] == "Ok"
       end
     end
 
     test "bad payment params" do
       with_mock HTTPoison,
         [request: fn(_method, _url, _body, _headers) -> MockResponse.bad_card_refund end] do
-          {:error, response} = ANet.refund(@amount, @refund_id, @opts_refund_bad_payment)
-          assert response.raw["ErrorResponse"]["messages"]["resultCode"] == "Error"
+          assert {:error, response} = ANet.refund(@amount, @refund_id, @opts_refund_bad_payment)
+          assert response.params["ErrorResponse"]["messages"]["resultCode"] == "Error"
       end
     end
 
     test "debit less than refund amount" do
```

Comment:
I'm repeating this, pls document this behavior in the `refund` docs. Also, could you rename the test? IMO the words debit and credit should never be used without mentioning whose account is being acted upon. Makes things ambiguous. Try to reword your docs too if there's any ambiguous use of debit or credit.

Reply:
I don't understand, how it is ambiguous? There are only two parties involved the `merchant` and the `customer`. Please comment if there is something I am missing.